components/frontend/src/header_footer/buttons/DownloadAsPDFButton.jsx

```diff
@@ -36,8 +36,11 @@ export function DownloadAsPDFButton({ report_uuid }) {
     const [loading, setLoading] = useState(false)
     // Make sure the report_url contains only registered query parameters
     const query = registeredURLSearchParams()
-    const queryString = query.toString() ? "?" + query.toString() : ""
-    query.set("report_url", window.location.origin + window.location.pathname + queryString + window.location.hash)
+    query.set("language", navigator.language)
+    query.set(
+        "report_url",
+        window.location.origin + window.location.pathname + "&" + query.toString() + window.location.hash,
```

Comment:
Does `pathname` always contain a `?` due to setting `navigator.language`? Seems odd to still be called `pathname` then

Reply:
Huh, there's indeed no `?`, but it does work. Apparently `&` instead of `?` works as well. But, I'll change it of course.

Reply:
Oh, I assumed that `query.set()` actually added a querystring to `location.pathname` somehow :D